spec/fixtures/activities/uksa/with_country.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There are multiple seed files as ingested the full file with 36 activities to test one specific condition was extra overhead. I made new files which are subsets of the original.